docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -54,21 +54,45 @@ Small changes might include:
 
 > **Note:** You can only edit a topic that is part of the latest release of documentation.
 
-## Before you begin
+### Before you begin
 
 - Create a [GitHub](https://www.github.com) account.
 
 **To edit a topic:**
 
 1. From a topic on the documentation website, click **Edit this page** (pencil icon).
-2. Enter your changes.
-3. Change the branch name, if required.
+1. Enter your changes.
+1. Change the branch name, if required.
    The branch name is auto-populated.
 
-4. Click **Propose changes**.
+1. Click **Propose changes**.
 
    A PR is created and then goes through the PR review and approval workflow.
 
+## Develop a new topic
+
+If you want to develop a new topic from scratch, you can create a documentation plan and collaborate with a member of the Docs team. According to the book [_Docs for Developers_](https://docsfordevelopers.com/), a documentation plan is a _flexible outline_ for anticipating where the writing process will lead you.
```

Comment:
"create a documentation plan"  I'm worried this will discourage community members and developers from contributing to the documentation.  Even as a technical writer, I don't write a content plan for every topic I write.  Other problems with this introduction:  1) How would they know how to contact the docs team?  2) this content assumes that the docs team has the bandwidth to consult on doc plans.

Reply:
All good points. Do you have an alternative advice for developers who want to create a new topic? Are you saying we don't want them to consult a writer at the outset of a process but to wait until it's already been drafted before reaching out to us?

Also the contact info for the doc team is given elsewhere in the doc. We could add it here, too.

Reply:
As much as I'd love to be consulted about everything people want to write?  That makes me the bottleneck for documentation.  I'd rather content get written if users need it.  We can always iterate on it later.

Reply:
It's true that there is an additional time ask of the technical writer: to look at the developer's doc plan and give feedback/advice based on the outline. However, I would make the argument that there is potentially much more time saved later on by by having fewer major corrections to make... and perhaps improvements in document quality. Personally, I would want to be consulted for.a quick sanity check before a big doc gets written, but I see that there is an argument for a different process to be made.

Reply:
I had a conversation with @krajorama today about where to put stuff for Helm chart documentation, and that is an example of far less work due to an up-front conversation. The earlier the ping the better, imo.

Reply:
deleted from this PR, will submit new PR in the future